apps/web/vibes/soul/sections/cart/index.tsx

```diff
@@ -378,90 +350,94 @@ function SubmitButton(props: React.ComponentPropsWithoutRef<typeof Button>) {
   return (
     <StickySidebarLayout
-      className="animate-pulse"
       sidebar={
-        <>
-          {/* Summary Title */}
-          <div className="mt-3.5 h-4 w-40 rounded-lg bg-contrast-100 @xl:h-7 @xl:w-52" />
-
-          {/* Subtotal */}
-          <div className="mt-[66px] flex justify-between border-b border-contrast-100/50 pb-5">
-            <div className="h-4 w-16 rounded-md bg-contrast-100" />
-            <div className="h-4 w-9 rounded-md bg-contrast-100" />
-          </div>
-
-          {/* Shipping */}
-          <div className="mt-5 flex justify-between border-b border-contrast-100/50 pb-5">
-            <div className="h-4 w-[70px] rounded-md bg-contrast-100" />
-            <div className="h-4 w-8 rounded-md bg-contrast-100" />
-          </div>
+        <div data-pending={pending ? '' : undefined}>
+          <h2 className="mb-10 font-heading text-4xl font-medium leading-none @xl:text-5xl">
+            {summaryTitle}
+          </h2>
+          <div className="w-full">
+            <div className="divide-y divide-contrast-100">
+              {Array.from({ length: summaryPlaceholderCount }).map((_, index) => (
+                <div className="py-4" key={index}>
+                  <div className="flex h-[1lh] w-full items-center justify-between">
+                    <div className="h-[1ch] w-16 rounded-md bg-contrast-100" />
+                    <div className="h-[1ch] w-9 rounded-md bg-contrast-100" />
+                  </div>
+                </div>
+              ))}
+            </div>
 
-          {/* Tax */}
-          <div className="mt-5 flex justify-between border-b border-contrast-100/50 pb-5">
-            <div className="h-4 w-8 rounded-md bg-contrast-100" />
-            <div className="h-4 w-8 rounded-md bg-contrast-100" />
+            <div className="flex justify-between border-t border-contrast-100 py-6 text-xl font-bold">
+              <div className="flex h-[1lh] w-full items-center">
+                <div className="h-[1ex] w-[5ch] rounded-md bg-contrast-100" />
+              </div>
+              <div className="flex h-[1lh] w-full items-center justify-end">
+                <div className="h-[1ex] w-[5ch] rounded-md bg-contrast-100" />
+              </div>
+            </div>
           </div>
 
-          {/* Grand Total */}
-          {/* <div className="mt-10 flex justify-between border-b border-contrast-100/50 pb-5">
-                <div className="h-6 w-20 rounded-lg bg-contrast-100" />
-                <div className="h-6 w-16 rounded-lg bg-contrast-100" />
-              </div> */}
-
-          {/* Checkout Button */}
-          <div className="mt-10 h-[50px] w-full rounded-full bg-contrast-100" />
-        </>
+          <div className="mt-4 h-[58px] w-full rounded-full bg-contrast-100" />
+        </div>
       }
       sidebarPosition="after"
       sidebarSize="1/3"
     >
-      <div className="w-full">
+      <div data-pending={pending ? '' : undefined}>
```

Comment:
We only need the `data-pending` at the top level

Reply:
I did this because the `StickySidebarLayout` component doesn't spread props. Do you think we should spread on that component?

Reply:
We only need it in 1 place and we're using it in the sidebar we're passing to `StickySidebarLayout`